neqo-client/src/main.rs

```diff
@@ -284,8 +284,8 @@ fn client(args: Args, socket: UdpSocket, local_addr: SocketAddr, remote_addr: So
 }
 
 fn main() {
+    init();
```

Comment:
So you aren't reading the database now?  That's OK until we need a client certificate.